Source/NSOutlineView.m

```diff
@@ -2198,10 +2204,113 @@ - (NSCell *) preparedCellAtColumn: (NSInteger)columnIndex row: (NSInteger)rowInd
 	  cell = [tb dataCellForRow: rowIndex];
 	}
     }
-  
+
   return cell;
 }
 
+- (NSRect) _drawOutlineCell: (NSTableColumn *)tb
+		       item: (id)item
+		drawingRect: (NSRect)inputRect
+		   rowIndex: (NSInteger)rowIndex
+{
+  NSRect drawingRect = inputRect;
+  NSImage *image = nil;
+  NSInteger level = 0;
+  CGFloat indentationFactor = 0.0;
+  CGFloat indentationPerLevel = [self indentationPerLevel];
+  NSCell *imageCell = nil;
+  NSRect imageRect;
+  id delegate = [self delegate];
+
+  // display the correct arrow...
+  if ([self isItemExpanded: item])
+    {
+      image = [NSImage imageNamed: @"common_ArrowDownH"];
```

Comment:
Here we have plenty of hard coded names and sizes, which would suggest to keep it in GSThemes.

Reply:
I thought you said this didn't belong in GSTheme above.   These images have always been hard-coded, I wish that they weren't.   I will move it back to GSTheme.  I'm wondering if we could change these images in the theme by providing alternate images.  Nevertheless, a discussion for later on.

Reply:
No, I did not. As I explained in the other discussion, you first reply to my comment was correct but later you seem to have mixed up things.
And yes, it is already possible to provide another image in a theme with that name. Still this code is a lot closer to themes as the other method where no images and border sizes show up.